.github/workflows/build.yml

```diff
@@ -13,14 +13,14 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu, macos]
-        java: ['8', '11', '17']
```

Comment:
Does the new asprof not work with java 8? or do they just not test it? I had an impression that  java 8 will be alive for ages?

Reply:
switched to zulu for now, shouldn't make a difference for building/testing